fetcher errors (TypeError, invalid-json)

After pulling the December changes, I'm seeing the following error multiple times right after MM2 startup:

```
[ERROR] Error initializing fetcher: TypeError: Cannot read property 'getStationName' of undefined
```

When the UI is loaded, the following errors appear:

```
[ERROR] FetchError: invalid json response body at https://bvg-apps-ext.hafas.de/bin/mgate.exe? reason: Unexpected end of JSON input
    at /home/magicmirror/MagicMirror/modules/MMM-PublicTransportBerlin/node_modules/node-fetch/lib/index.js:273:32
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async BvgFetcher.getStationName (/home/magicmirror/MagicMirror/modules/MMM-PublicTransportBerlin/BvgFetcher.js:27:19)
    at async Class.createFetcher (/home/magicmirror/MagicMirror/modules/MMM-PublicTransportBerlin/node_helper.js:22:27) {
  type: 'invalid-json'
}
```

--- a/BvgFetcher.js
+++ b/BvgFetcher.js
@@ -28,7 +28,7 @@ class BvgFetcher {
 
   async getDirectionDescriptor() {
     if (typeof this.config.directionStationId === "undefined") {
-      return "all directions"
+      return "all directions";
     } else {
       const station = await bvgClient.stop(this.config.directionStationId);
       return station.name;
@@ -53,13 +53,13 @@ class BvgFetcher {
     if (!this.config.directionStationId || this.config.directionStationId === "") {
       opt = {
         when: when,
-        duration: this.config.departureMinutes,
+        duration: this.config.departureMinutes
       };
     } else {
       opt = {
         direction: this.config.directionStationId,
         when: when,
-        duration: this.config.departureMinutes,
+        duration: this.config.departureMinutes
       };
     }
 
@@ -72,7 +72,7 @@ class BvgFetcher {
   processData(data) {
     let departuresData = {
       fetcherId: this.id,
-      departuresArray: [],
+      departuresArray: []
     };
 
     data.forEach((row) => {
@@ -97,7 +97,7 @@ class BvgFetcher {
           name: row.line.name,
           nr: row.line.nr,
           type: row.line.product,
-          direction: row.direction,
+          direction: row.direction
         };
 
         departuresData.departuresArray.push(current);

--- a/BvgFetcher.js
+++ b/BvgFetcher.js
@@ -4,9 +4,7 @@ const bvgProfile = require("hafas-client/p/bvg");
 const pjson = require("./package.json");
 const bvgClient = createClient(
   bvgProfile,
-  "MagicMirror2 module MMM-PublicTransportBerlin v" +
-    pjson.version +
-    " (https://github.com/deg0nz/MMM-PublicTransportBerlin)"
+  "MMM-PublicTransportBerlin v" + pjson.version
 );
 
 class BvgFetcher {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "MMM-PublicTransportBerlin",
-  "version": "1.6.3",
+  "version": "1.6.4",
   "description": "BVG-Hafas driven public transport module for MagicMirror 2.",
   "repository": {
     "type": "git",

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
     "node": ">=10"
   },
   "dependencies": {
-    "hafas-client": "^5.22.1",
+    "hafas-client": "^5.22.2",
     "moment": "^2.29.1",
     "vbb-line-colors": "^1.0.4",
     "vbb-short-station-name": "^1.0.1"

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
     "node": ">=10"
   },
   "dependencies": {
-    "hafas-client": "^5.22.0",
+    "hafas-client": "^5.22.1",
     "moment": "^2.29.1",
     "vbb-line-colors": "^1.0.4",
     "vbb-short-station-name": "^1.0.1"

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "MMM-PublicTransportBerlin",
   "version": "1.6.4",
-  "description": "BVG-Hafas driven public transport module for MagicMirror 2.",
+  "description": "Public transport module for MagicMirror² driven by BVG-Hafas data.",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/deg0nz/MMM-PublicTransportBerlin"

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
     "email": "magicmirror@kayuk.de",
     "url": "https://kayuk.de"
   },
-  "contributors": "https://github.com/deg0nz/MMM-PublicTransportBerlin/graphs/contributors",
+  "contributors": ["https://github.com/deg0nz/MMM-PublicTransportBerlin/graphs/contributors"],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/deg0nz/MMM-PublicTransportBerlin/issues"

